vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -33,6 +35,17 @@ type internal FSharpQuickInfoProvider
 
     let xmlMemberIndexService = serviceProvider.GetService(typeof<SVsXMLMemberIndexService>) :?> IVsXMLMemberIndexService
     let documentationBuilder = XmlDocumentation.CreateDocumentationBuilder(xmlMemberIndexService, serviceProvider.DTE)
+    let navigateTo (workspace: Workspace) (range: range)  = async {
+        let documentNavigationService =workspace.Services.GetService<IDocumentNavigationService>()
+        let filePath = System.IO.Path.GetFullPathSafe range.FileName
+        match workspace.CurrentSolution.GetDocumentIdsWithFilePath(filePath) |> List.ofSeq with
+        | id :: _ ->
```

Comment:
If file is linked to several projects, it's possible to jump to the wrong document. We need to find a document for which  `FSharpSymbol.Assembly.SimpleName` = `document.Project.AssemblyName`.